main/src/main/java/org/mobilitydata/gtfsvalidator/validator/PathwayStopsConsistencyValidator.java

```diff
@@ -82,7 +82,7 @@ public void validate(NoticeContainer noticeContainer) {
   /**
    * A stop referenced by a row from {@code pathways.txt} has {@code location_type = 1} (station).
    *
-   * <p>Severity: {@code SeverityLevel.WARNING} to be upgraded to {@code SeverityLevel.ERROR}.
+   * <p>Severity: {@code SeverityLevel.ERROR}
```

Comment:
This validator was removed in pull #978

You just need to sync to the latest code.